lib/github/github_service.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+module Github
+  class GithubService
+    class << self
+      include SentryLogging
+      extend Memoist
+
+      # TODO : use actual repo
+      GITHUB_REPO = 'thebravery/thebravery_project'
+
+      def create_issue(feedback)
+        # feedback comes in as a hash, convert to model
```

Comment:
:+1: 
It tripped me up for a minute because it would be a `#<Feedback:0xblahablah>` model object but since it gets stored to Redis before the async Sidekiq worker picks it up, it's coerced to a `hash`.